featuretools/tests/primitive_tests/test_rolling_primitive.py

```diff
@@ -420,3 +421,49 @@ def test_rolling_trend_non_uniform():
         primitive_instance(no_freq_series.index, pd.Series(no_freq_series.values)),
     )
     pd.testing.assert_series_equal(rolled_series, expected_series)
+
+
+@pytest.mark.parametrize(
+    "window_length, gap",
+    [
+        (5, 2),
+        (5, 0),
+        ("5d", "7d"),
+        ("5d", "0d"),
+    ],
+)
+@pytest.mark.parametrize(
+    "min_periods",
+    [1, 0, 2, 5],
+)
+def test_rolling_outlier_count(
+    min_periods,
+    window_length,
+    gap,
+    rolling_outlier_series_pd,
+):
+
+    primitive_instance = RollingOutlierCount(
+        window_length=window_length,
+        gap=gap,
+        min_periods=min_periods,
+    )
+
+    primitive_func = primitive_instance.get_function()
+
+    actual_vals = pd.Series(
+        primitive_func(
+            rolling_outlier_series_pd.index,
+            pd.Series(rolling_outlier_series_pd.values),
+        ),
+    )
+
+    expected_vals = apply_rolling_agg_to_series(
+        series=rolling_outlier_series_pd,
+        agg_func=primitive_instance.get_outliers_count,
+        window_length=window_length,
+        gap=gap,
+        min_periods=min_periods,
+    )
+
+    pd.testing.assert_series_equal(actual_vals, pd.Series(data=expected_vals))
```

Comment:
Is there a reason the nan check that the other primitive tests do wasn't included?

Reply:
Added it in!